platform/core/src/services/ViewerToolsetService/ViewerToolset.js

```diff
@@ -4,16 +4,16 @@ const publicAPI = {
   name,
   getState: _getState,
   setIsReferenceLinesEnabled: _setIsReferenceLinesEnabled,
-  setIsCrosshairsEnabled: _setIsCrosshairsEnabled,
+  setIsSeriesLinkingEnabled: _setIsSeriesLinkingEnabled,
   setServiceImplementation,
 };
 
 const serviceImplementation = {
   _getState: () => console.warn('getState() NOT IMPLEMENTED'),
   _setIsReferenceLinesEnabled: () =>
     console.warn('setIsReferenceLinesEnabled() NOT IMPLEMENTED'),
-  _setIsCrosshairsEnabled: () =>
-    console.warn('setIsCrosshairsEnabled() NOT IMPLEMENTED'),
+  _setIsSeriesLinkingEnabled: () =>
```

Comment:
Same here:  Why is setIsCrosshairsEnabled removed now and replaced with series linking. Applies to rest of th e PR too